firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FeedbackActivity.java

```diff
@@ -30,10 +35,9 @@
 
 /** Activity for tester to compose and submit feedback. */
 public class FeedbackActivity extends AppCompatActivity {
-
   private static final String TAG = "FeedbackActivity";
-  private static final int THUMBNAIL_WIDTH = 200;
-  private static final int THUMBNAIL_HEIGHT = 200;
+  private static final int THUMBNAIL_WIDTH = 600;
```

Comment:
Nit: include units in the name?  I can't tell if this is pixels or percent or something.

Reply:
DONE